docker-compose/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
     volumes:
       - $NATS_SURVEYOR_CREDS:/etc/surveyor/SYS.creds
       - ./observations:/observations
+      - ./jetstream:/jetstream
```

Comment:
I'm wondering if the storage here should be consolidated.  One volume with obervations.json, jetstream.json, etc, or even consolidate into one config file.  wdyt?